web_src/js/features/repo-legacy.js

```diff
@@ -439,7 +439,8 @@ async function onEditContent(event) {
 
   if (!$editContentZone.html()) {
     $editContentZone.html($('#issue-comment-editor-template').html());
-    comboMarkdownEditor = await initComboMarkdownEditor($editContentZone.find('.combo-markdown-editor'));
+    const comboMarkdownEditors = await initComboMarkdownEditor($editContentZone.find('.combo-markdown-editor'));
+    comboMarkdownEditor = comboMarkdownEditors[0];
```

Comment:
By design, `initComboMarkdownEditor` only initializes one editor.

Otherwise there will be fragile code like:


![image](https://user-images.githubusercontent.com/2114189/235424084-6c31c471-6652-44e7-b7ad-ed79245bb828.png)

Reply:
Yes, remove these `[0]` usage.

Reply:
use `find().each()` to init all combo markdown editor.